master/buildbot/reporters/github.py

```diff
@@ -205,5 +216,6 @@ def createStatus(self,
         payload = {'body': description}
 
         return self._http.post(
-            '/'.join(['/repos', repo_user, repo_name, 'issues', issue, 'comments']),
+            '/'.join(['/repos', repo_user, repo_name,
```

Comment:
What is the goal of such wrapping here, and leave [this one](https://github.com/dragon512/buildbot/blob/bb83f28def75730132d4d6709c0630009e70dd1d/master/buildbot/reporters/github.py#L188) as is? (Since it is less 80 characters anyway)